test/testServer.js

```diff
@@ -189,6 +210,25 @@ app.post('/:id/query', function(req, res) {
                 "sid": "dunnowhatthisis"
             };
             break;
+        case 'testNumberReset':
+            console.log('Here');
```

Comment:
Hi Lawrie, do you still need this console.log statement and that on line 293?  Thanks.